fetch-quotes.test.js

```diff
@@ -0,0 +1,9 @@
+import fetchQuotes from './fetch-quotes';
+
+describe('fetchQuotes', () => {
+  it('fetches a single quote object from API', async () => {
+    const quote = await fetchQuotes();
+      
+    expect(quote).toEqual(expect.any(Object));
```

Comment:
Think you should separate this more.  Basically, this will always pass, no matter what kind of object you pass or array.   

That being said, you can break this up to be more specific.  Like having an obj that has { name: expect.any(String) etc etc}.  That way, you know you want an object with a key and value of name, text, etc.  Let me know if that makes sense